tempodb/backend/s3/s3.go

```diff
@@ -660,6 +660,8 @@ func createCore(cfg *Config, hedge bool) (*minio.Core, error) {
 		opts.BucketLookup = minio.BucketLookupType(cfg.BucketLookupType)
 	}
 
+	opts.UseDualStack = cfg.UseDualStack
```

Comment:
This option does not exist. Disabling dualstack endpoints is configured via a method in the client https://github.com/minio/minio-go/pull/1945/files#diff-f7bb71ca856c07b3329265f26b1e577e00b71e4792550cf14346bdbe0e7bd18dR342-R346.

Reply:
This is why the pipeline is failing

Reply:
So in this case, according to [this](https://arc.net/l/quote/vhuzymsw) function, the minio is going to use the dualstack endpoint everytime there's an s3 url present, right ?. How can we override this ?

Reply:
You just need to call that function with `false`